examples/misc/keyword_plotting.py

```diff
@@ -0,0 +1,28 @@
+"""
+======================
+Plotting with keywords
+======================
+
+There are some instances where you have data in a format that lets you
+access particular variables with strings. For example, with
+:class:`numpy.recarray` or :class:`pandas.DataFrame`.
+
+Matplotlib allows you provide such an object with
+the ``data`` keyword argument. If provided, then you may generate plots with
```

Comment:
oh - ok I removed the extra whitespace. is that such a big deal? I prefer to have extra whitespace as it makes it easier to reword etc without having to bump everything down a bunch of lines